mw-webapp/package.json

```diff
@@ -132,5 +99,38 @@
     "vite": "5.2.11",
     "vite-plugin-eslint": "1.8.1",
     "vite-tsconfig-paths": "4.3.2"
+  },
+  "engines": {
+    "node": "20.11.0",
+    "pnpm": "8.15.2"
+  },
```

Comment:
I see the same engines requirements (and packageManager, check for other) in the root package.json, do we need duplication?

Reply:
done